pyproject.toml

```diff
@@ -43,16 +43,21 @@ onnx = [
     "**/*.h",
     "**/*.proto",
     "**/*.pyi",
-    "backend/test/data/**/*",
     "py.typed",
 ]
 
+[tool.setuptools.exclude-package-data]
```

Comment:
I think the change goes into the right direction. My only concern is runtimes such as onnxruntime expects to find this data. I'd me more confortable to display an error message saying what instruction is needed to get the data and have the backend tests passing.

Reply:
onnxruntime already fetches the entire onnx source during the build process anyway. Are they really using the PyPI package for testing, too?

Reply:
Yes, it is used. onnx should be shipped without data. That's not a debate. Since this data can be automatically generated, we could also generate this data when the package is being installed. Something like ``pip install onnx <options>``... So developers using the data could just install onnx with this option and nothing else would change.